docs/core/tools/dotnet-list-package.md

```diff
@@ -0,0 +1,101 @@
+---
+title: dotnet list package command
+description: The 'dotnet list package' command provides a convenient option to list the package references for a project or solution.
+ms.date: 01/30/2019
+---
+# dotnet list package
+
+[!INCLUDE [topic-appliesto-net-core-22plus](../../../includes/topic-appliesto-net-core-22plus.md)]
+
+## Name
+
+`dotnet list package` - Lists the package references for a project or solution.
+
+## Synopsis
+
+```
+dotnet list [<PROJECT | SOLUTION>] package [--config] [--framework] [--highest-minor] [--highest-patch] [--include-prerelease] [--include-transitive] [--outdated] [--source]
+dotnet list package [-h|--help]
+```
+
+## Description
+
+The `dotnet list package` command provides a convenient option to list all NuGet package references for a specific project or a solution. You first need to build the project in order to have the assets needed for this command to run. The following example shows the output of the `dotnet list package` command for the [SentimentAnalysis](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/SentimentAnalysis) project:
+
+```output
+Project 'SentimentAnalysis' has the following package references
+   [netcoreapp2.1]:
+   Top-level Package               Requested   Resolved
+   > Microsoft.ML                  0.9.0       0.9.0
+   > Microsoft.NETCore.App   (A)   [2.1.0, )   2.1.0
+
+(A) : Auto-referenced package.
+```
+
+The **Requested** column refers to the package version specified in the project file and can be a range. The **Resolved** column lists the version that the project is currently using and is always a single value. The packages displaying an `(A)` right next to their names represent [implicit package references](csproj.md#implicit-package-references) that are inferred from your project settings (`Sdk` type, `<TargetFramework>` or `<TargetFrameworks>` property, etc.)
+
+Use the `--outdated` option to find out if there are newer versions available of the packages you're using on your projects. By default, `--outdated` lists the latest stable packages unless the resolved version is also a prerelease version. To include prerelease versions when listing newer versions, also specify the `--include-prerelease` option.
+
+## Arguments
+
+`PROJECT | SOLUTION`
+
+The project or solution file to operate on. If not specified, the command searches the current directory for one. If more than one solution or project is found, an error is thrown.
+
+## Options
+
+* **`--config <SOURCE>`**
+
+  The NuGet sources to use when searching for newer packages. Requires the `--outdated` option.
+
+* **`--framework <FRAMEWORK | FRAMEWORK/RID>`**
```

Comment:
We should check the latest 3.0 build to see if RID is still accepted. I know we did some work to remove RID from the output, since the packages should always be the same as the framework without the RID.

Reply:
I have a work item to review the CLI as a whole for 3.0. Who would know about these changes?

Reply:
If RIDs are accepted, is there a difference between \<FRAMEWORK | FRAMEWORK/RID>? Isn't \<FRAMEWORK/RID> adequate?

Reply:
The format is FRAMEWORK/RID like `netstandard2.0/win-x64`

Reply:
I had a quick chat with my team about it today. Previously (currently for GA versions) there are times when the SDK reports both TFM and TFM/RID to NuGet, causing NuGet to list the same packages list twice. This has been removed from the upcoming version, so NuGet will only report the TFM once. However, we didn't make any changes to the input parsing. So, while you can ask for `--framework netcoreapp2.0/win-x64`, it will either output netcoreapp2.0 packages without the win-x64 RID, or it won't output any packages at all. We need to add a work item to out backlog to remove the possibility of specifying RID as an input parameter. But for now the CLI allows you to specify `netcoreapp2.0/win-x64`, so I leave it up to your judgement if the docs should say this for now.

Reply:
I think it's not an idea use-case, I'll just remove from the docs and keep just the framework option.